### DISCUSSION TITLE
Add control to canvas in code and the control's resizing is shaking

### DISCUSSION THREAD

**Discussion by jeff9571** [AvaloniaUI/Avalonia#15739] [Q&A] [answered]
### Description
Two buttons in a canvas, the **Button** is created in code, the **Adorned Button** is defined in axaml as a child of the canvas. When I resize the adorner, the button created in code seems like it's shaking, and the other button defined in xaml works fine. How to make the dynamically created button act like the axaml button?

### Behavior
https://github.com/AvaloniaUI/Avalonia/assets/35749706/d1ab7f30-04d5-406d-ae95-d82f30e35e4d

### Code
xaml button
![image](https://github.com/AvaloniaUI/Avalonia/assets/35749706/a1461faf-052f-475b-935b-b411d66bd7b5)

button created in code
![image](https://github.com/AvaloniaUI/Avalonia/assets/35749706/95aa9436-0b4b-4782-b638-9eac2d9df6f1)

**Comment by maxkatz6** [comment, marked as answer]
Most likely, you need to set UseLayoutRounding to false on canvas.

**Reply by jeff9571** [reply]
It works, thanks.
